apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transactions.ex

```diff
@@ -153,6 +153,7 @@ defmodule EthereumJSONRPC.Transactions do
   """
   def to_elixir(transactions, block_timestamp \\ nil) when is_list(transactions) do
     transactions
+    |> Enum.filter(fn tx -> tx["type"] != "0x2" end)
     |> Enum.map(&Transaction.to_elixir(&1, block_timestamp))
```

Comment:
We don't need to filter all transactions by type 0x2, we need to define transactions of type 0x2 and store additional fields for them. We may need to display them as a separate list in the interface, so we may need the ability to filter them in the API part.